site/_docs/history.md

```diff
@@ -28,23 +28,50 @@ For a full list of releases, see
 Downloads are available on the
 [downloads page]({{ site.baseurl }}/downloads/).
 
-<!--
-## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.34.0">1.34.0</a> / XXXX-XX-XX
+## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.34.0">1.34.0</a> / 2023-03-XX
 {: #v1-34-0}
 
-This release comes x months after [1.33.0](#v1-33-0),
-contains contributions from x contributors, and resolves x issues.
+This release comes 1 month after [1.33.0](#v1-33-0),
+contains contributions from 18 contributors, and resolves 34 issues. It adds roughly 15 new functions
+in BigQuery library for handling dates, times, and timestamps and provides a fix
+([<a href="https://issues.apache.org/jira/browse/CALCITE-5522">CALCITE-5522</a>]) for a small breaking
+change in `DATE_TRUNC` function ([<a href="https://issues.apache.org/jira/browse/CALCITE-5447">CALCITE-5447</a>]),
+which was introduced accidentally in [1.33.0](#v1-33-0). Among other small improvements and fixes it
+brings a new breaking change ([<a href="https://issues.apache.org/jira/browse/CALCITE-3870">CALCITE-3870</a>])
+changing the default sub-query expansion strategy of `SqlToRelConverter`. 
```

Comment:
I don't think that 3870 needs a mention here, as it's covered below. But QUALIFY (5268) is a major new feature.

Reply:
Fixed in https://github.com/apache/calcite/pull/3103/commits/c17c0c7f1c4254632d87f21b860849bea6551cdd